### PR TITLE
chore: get real committer name && optimise dockerfile

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve? 

as title

### What is changed and how it works?

* get committer from github.event.puhser.name
* remove mydmber from docker image
* add `dm-master/dm-worker/dmctl` to image `/bin`

### Check List 

Tests 
 - Manual test (add detailed scripts or steps below)
     - Manually test ci [here](https://github.com/Ehco1996/dm/runs/3129916613?check_suite_focus=true)

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,9 +20,14 @@ RUN make dm-worker dm-master dmctl
 
 FROM alpine:3.10
 
+# keep compatibility
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-worker /dm-worker
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-master /dm-master
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dmctl /dmctl
-COPY --from=builder /opt/tools/bin/mydumper /mydumper
```
Comment (lance6716) [inline Dockerfile:31]: `make check` may need this binary, please check if CI could work

Comment (Ehco1996) [reply Dockerfile:31]: now dm not need this bin, docker image already build here https://github.com/pingcap/dm/pull/1902/checks?check_run_id=3129978217

**Review by lance6716 — APPROVED** [review]

**Review by glorv — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -25,9 +20,14 @@ RUN make dm-worker dm-master dmctl
...
 
-EXPOSE 8291 8261 8262
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-worker /bin/dm-worker
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-master /bin/dm-master
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dmctl /bin/dmctl
+
```
Comment (GMHDBJD) [inline Dockerfile:31]: Why change binary dir

Comment (GMHDBJD) [reply Dockerfile:31]: dm-operator or users may use origin binary dir in script.

Comment (Ehco1996) [reply Dockerfile:31]: origin binary is keeped, i just make another copy to /bin/ so it’s easier for users to use when exec pod

**Pull request merged**